### ISSUE TITLE
java.lang.RuntimeException: Attempted to load class net/minecraft/client/world/ClientWorld for invalid dist DEDICATED_SERVER

### ISSUE THREAD

**Issue by ZL1LAC** [MuradAkh/LittleLogistics#4]
Unable to load a server running forge-1.16.5-36.2.22.

```
---- Minecraft Crash Report ----
// Ouch. That hurt :(

Time: 1/12/22 9:15 AM
Description: Mod loading error has occurred

java.lang.Exception: Mod Loading has failed
	at net.minecraftforge.fml.CrashReportExtender.dumpModLoadingCrashReport(CrashReportExtender.java:85) ~[forge:?] {re:classloading}
	at net.minecraftforge.fml.server.ServerModLoader.load(ServerModLoader.java:51) ~[forge:?] {re:classloading}
	at net.minecraft.server.Main.main(Main.java:95) ~[?:?] {re:classloading}
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_292] {}
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_292] {}
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_292] {}
	at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_292] {}
	at net.minecraftforge.fml.loading.FMLServerLaunchProvider.lambda$launchService$0(FMLServerLaunchProvider.java:51) ~[forge-1.16.5-36.2.22.jar:36.2] {}
	at cpw.mods.modlauncher.LaunchServiceHandlerDecorator.launch(LaunchServiceHandlerDecorator.java:37) [modlauncher-8.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:54) [modlauncher-8.0.9.jar:?] {}
	at cpw.mods.modlauncher.LaunchServiceHandler.launch(LaunchServiceHandler.java:72) [modlauncher-8.0.9.jar:?] {}
	at cpw.mods.modlauncher.Launcher.run(Launcher.java:82) [modlauncher-8.0.9.jar:?] {re:classloading}
	at cpw.mods.modlauncher.Launcher.main(Launcher.java:66) [modlauncher-8.0.9.jar:?] {re:classloading}
	at net.minecraftforge.server.ServerMain$Runner.runLauncher(ServerMain.java:63) [forge-1.16.5-36.2.22.jar:?] {}
	at net.minecraftforge.server.ServerMain$Runner.access$100(ServerMain.java:60) [forge-1.16.5-36.2.22.jar:?] {}
	at net.minecraftforge.server.ServerMain.main(ServerMain.java:57) [forge-1.16.5-36.2.22.jar:?] {}


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: main
Stacktrace:
	at net.minecraftforge.fml.loading.RuntimeDistCleaner.processClassWithFlags(RuntimeDistCleaner.java:71) ~[forge-1.16.5-36.2.22.jar:36.2] {}
-- MOD shipping --
Details:
	Mod File: shipping-pre-alpha-1.0.1.jar
	Failure message: Little Logistics (shipping) has failed to load correctly
		java.lang.RuntimeException: Attempted to load class net/minecraft/client/world/ClientWorld for invalid dist DEDICATED_SERVER
	Mod Version: pre-alpha-1.0.1
	Mod Issue URL: NOT PROVIDED
	Exception message: java.lang.RuntimeException: Attempted to load class net/minecraft/client/world/ClientWorld for invalid dist DEDICATED_SERVER
Stacktrace:
	at net.minecraftforge.fml.loading.RuntimeDistCleaner.processClassWithFlags(RuntimeDistCleaner.java:71) ~[forge-1.16.5-36.2.22.jar:36.2] {}
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:85) ~[modlauncher-8.0.9.jar:?] {}
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-8.0.9.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader$DelegatedClassLoader.findClass(TransformingClassLoader.java:265) ~[modlauncher-8.0.9.jar:?] {}
	at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:136) ~[modlauncher-8.0.9.jar:?] {re:classloading}
	at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:98) ~[modlauncher-8.0.9.jar:?] {re:classloading}
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_292] {}
	at java.lang.Class.getDeclaredMethods0(Native Method) ~[?:1.8.0_292] {}
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701) ~[?:1.8.0_292] {}
	at java.lang.Class.getDeclaredMethod(Class.java:2128) ~[?:1.8.0_292] {}
	at net.minecraftforge.eventbus.EventBus.getDeclMethod(EventBus.java:83) ~[eventbus-4.0.0.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.lambda$null$4(EventBus.java:95) ~[eventbus-4.0.0.jar:?] {}
	at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:193) ~[?:1.8.0_292] {}
	at java.util.HashMap$KeySpliterator.tryAdvance(HashMap.java:1577) ~[?:1.8.0_292] {}
	at java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:126) ~[?:1.8.0_292] {}
	at java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:499) ~[?:1.8.0_292] {}
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:486) ~[?:1.8.0_292] {}
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472) ~[?:1.8.0_292] {}
	at java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:152) ~[?:1.8.0_292] {}
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_292] {}
	at java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:531) ~[?:1.8.0_292] {}
	at net.minecraftforge.eventbus.EventBus.lambda$registerObject$7(EventBus.java:97) ~[eventbus-4.0.0.jar:?] {}
	at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[?:1.8.0_292] {}
	at java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:175) ~[?:1.8.0_292] {}
	at java.util.Spliterators$ArraySpliterator.forEachRemaining(Spliterators.java:948) ~[?:1.8.0_292] {}
	at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:482) ~[?:1.8.0_292] {}
	at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:472) ~[?:1.8.0_292] {}
	at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:1.8.0_292] {}
	at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:1.8.0_292] {}
	at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:1.8.0_292] {}
	at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:485) ~[?:1.8.0_292] {}
	at net.minecraftforge.eventbus.EventBus.registerObject(EventBus.java:94) ~[eventbus-4.0.0.jar:?] {}
	at net.minecraftforge.eventbus.EventBus.register(EventBus.java:120) ~[eventbus-4.0.0.jar:?] {}
	at dev.murad.shipping.ShippingMod.<init>(ShippingMod.java:55) ~[shipping:pre-alpha-1.0.1] {re:classloading}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method) ~[?:1.8.0_292] {}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62) ~[?:1.8.0_292] {}
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45) ~[?:1.8.0_292] {}
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423) ~[?:1.8.0_292] {}
	at java.lang.Class.newInstance(Class.java:442) ~[?:1.8.0_292] {}
	at net.minecraftforge.fml.javafmlmod.FMLModContainer.constructMod(FMLModContainer.java:81) ~[forge:36.2] {re:classloading}
	at net.minecraftforge.fml.ModContainer.lambda$buildTransitionHandler$4(ModContainer.java:120) ~[forge:?] {re:classloading}
	at java.util.concurrent.CompletableFuture$AsyncRun.run(CompletableFuture.java:1640) ~[?:1.8.0_292] {}
	at java.util.concurrent.CompletableFuture$AsyncRun.exec(CompletableFuture.java:1632) ~[?:1.8.0_292] {}
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289) ~[?:1.8.0_292] {}
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056) ~[?:1.8.0_292] {}
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692) ~[?:1.8.0_292] {}
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:175) ~[?:1.8.0_292] {}


-- System Details --
Details:
	Minecraft Version: 1.16.5
	Minecraft Version ID: 1.16.5
	Operating System: Linux (amd64) version 4.15.0-163-generic
	Java Version: 1.8.0_292, AdoptOpenJDK
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), AdoptOpenJDK
	Memory: 1981032792 bytes (1889 MB) / 2506096640 bytes (2390 MB) up to 5726797824 bytes (5461 MB)
	CPUs: 16
	JVM Flags: 1 total; -Xmx6144M
	ModLauncher: 8.0.9+86+master.3cf110c
	ModLauncher launch target: fmlserver
	ModLauncher naming: srg
	ModLauncher services: 
		/mixin-0.8.4.jar mixin PLUGINSERVICE 
		/eventbus-4.0.0.jar eventbus PLUGINSERVICE 
		/forge-1.16.5-36.2.22.jar object_holder_definalize PLUGINSERVICE 
		/forge-1.16.5-36.2.22.jar runtime_enum_extender PLUGINSERVICE 
		/accesstransformers-3.0.1.jar accesstransformer PLUGINSERVICE 
		/forge-1.16.5-36.2.22.jar capability_inject_definalize PLUGINSERVICE 
		/forge-1.16.5-36.2.22.jar runtimedistcleaner PLUGINSERVICE 
		/mixin-0.8.4.jar mixin TRANSFORMATIONSERVICE 
		/forge-1.16.5-36.2.22.jar fml TRANSFORMATIONSERVICE 
	FML: 36.2
	Forge: net.minecraftforge:36.2.22
	FML Language Providers: 
		javafml@36.2
		minecraft@1
	Mod List: 
		rsrequestify-1.16.5-2.1.6.jar                     |RSRequestify                  |rsrequestify                  |2.1.6               |CREATE_REG|Manifest: NOSIGNATURE
		CyclopsCore-1.16.5-1.12.1.jar                     |Cyclops Core                  |cyclopscore                   |1.12.1              |CREATE_REG|Manifest: NOSIGNATURE
		astralsorcery-1.16-1.16.5-1.13.12.jar             |Astral Sorcery                |astralsorcery                 |1.16.5-1.13.12      |CREATE_REG|Manifest: 45:2b:0a:49:6b:65:3b:39:a9:dd:d2:5b:55:7f:82:47:a5:1d:7a:cc:7f:a8:69:73:72:53:6f:57:4d:b2:1a:b7
		ftb-essentials-1605.1.5-build.32.jar              |FTB Essentials                |ftbessentials                 |1605.1.5-build.32   |CREATE_REG|Manifest: NOSIGNATURE
		ProjectE-1.16.5-PE1.0.1B.jar                      |ProjectE                      |projecte                      |PE1.0.1B            |CREATE_REG|Manifest: NOSIGNATURE
		Neat 1.7-27.jar                                   |Neat                          |neat                          |1.7-27              |CREATE_REG|Manifest: NOSIGNATURE
		CTM-MC1.16.1-1.1.2.6.jar                          |ConnectedTexturesMod          |ctm                           |MC1.16.1-1.1.2.6    |CREATE_REG|Manifest: NOSIGNATURE
		CookingForBlockheads_1.16.5-9.3.4.jar             |Cooking for Blockheads        |cookingforblockheads          |9.3.4               |CREATE_REG|Manifest: NOSIGNATURE
		ReAuth-1.16-Forge-3.9.3.jar                       |ReAuth                        |reauth                        |3.9.3               |CREATE_REG|Manifest: 3d:06:1e:e5:da:e2:ff:ae:04:00:be:45:5b:ff:fd:70:65:00:67:0b:33:87:a6:5f:af:20:3c:b6:a1:35:ca:7e
		Placebo-1.16.5-4.6.1.jar                          |Placebo                       |placebo                       |4.6.1               |CREATE_REG|Manifest: NOSIGNATURE
		citadel-1.8.1-1.16.5.jar                          |Citadel                       |citadel                       |1.8.1               |CREATE_REG|Manifest: NOSIGNATURE
		serversync-forge-loader-2.1.0.jar                 |ServerSync Forge Loader       |serversync-forge-loader       |2.1.0               |CREATE_REG|Manifest: NOSIGNATURE
		Powah-1.16.5-2.3.16.jar                           |Powah                         |powah                         |2.3.16              |CREATE_REG|Manifest: NOSIGNATURE
		rangedpumps-0.8.2.jar                             |Ranged Pumps                  |rangedpumps                   |0.8.2               |CREATE_REG|Manifest: NOSIGNATURE
		culinaryconstruct-forge-1.16.5-4.0.0.6.jar        |Culinary Construct            |culinaryconstruct             |1.16.5-4.0.0.6      |CREATE_REG|Manifest: NOSIGNATURE
		Bookshelf-Forge-1.16.5-10.3.29.jar                |Bookshelf                     |bookshelf                     |10.3.29             |CREATE_REG|Manifest: eb:c4:b1:67:8b:f9:0c:db:dc:4f:01:b1:8e:61:64:39:4c:10:85:0b:a6:c4:c7:48:f0:fa:95:f2:cb:08:3a:e5
		buildinggadgets-1.16.5-3.8.2.jar                  |Building Gadgets              |buildinggadgets               |3.8.2               |CREATE_REG|Manifest: NOSIGNATURE
		Morpheus-1.16.5-4.2.70.jar                        |Morpheus                      |morpheus                      |4.2.70              |CREATE_REG|Manifest: NOSIGNATURE
		Hwyla-forge-1.10.11-B78_1.16.2.jar                |Waila                         |waila                         |1.10.11-B78_1.16.2  |CREATE_REG|Manifest: NOSIGNATURE
		FpsReducer-forge-1.24-mc1.16.5.jar                |FPS Reducer                   |fpsreducer                    |1.24-mc1.16.5       |CREATE_REG|Manifest: NOSIGNATURE
		JustEnoughResources-1.16.5-0.12.1.133.jar         |Just Enough Resources         |jeresources                   |0.12.1.133          |CREATE_REG|Manifest: NOSIGNATURE
		MmmMmmMmmMmm-1.16.5-1.3.1.jar                     |MmmMmmMmmMmm                  |dummmmmmy                     |1.3.0               |CREATE_REG|Manifest: NOSIGNATURE
		twilightforest-1.16.5-4.0.870-universal.jar       |The Twilight Forest           |twilightforest                |NONE                |CREATE_REG|Manifest: NOSIGNATURE
		refinedstorage-1.9.16.jar                         |Refined Storage               |refinedstorage                |1.9.16              |CREATE_REG|Manifest: NOSIGNATURE
		PackMenu-1.16.5-2.5.0.jar                         |PackMenu                      |packmenu                      |2.5.0               |CREATE_REG|Manifest: NOSIGNATURE
		industrial-foregoing-1.16.5-3.2.14.6-14.jar       |Industrial Foregoing          |industrialforegoing           |3.2.14.6            |CREATE_REG|Manifest: NOSIGNATURE
		aequivaleo-0.1.75-BETA-universal.jar              |Aequivaleo                    |aequivaleo                    |0.1.75-BETA         |CREATE_REG|Manifest: NOSIGNATURE
		torchmaster-2.3.8.jar                             |Torchmaster                   |torchmaster                   |2.3.8               |CREATE_REG|Manifest: NOSIGNATURE
		CommonCapabilities-1.16.5-2.8.0.jar               |CommonCapabilities            |commoncapabilities            |2.8.0               |CREATE_REG|Manifest: NOSIGNATURE
		valkyrielib-1.16.5-3.0.9.5.jar                    |ValkyrieLib                   |valkyrielib                   |1.16.5-3.0.9.5      |CREATE_REG|Manifest: NOSIGNATURE
		Lollipop-1.16.5-3.2.9.jar                         |Lollipop                      |lollipop                      |3.2.9               |CREATE_REG|Manifest: NOSIGNATURE
		Botania-1.16.5-420.1.jar                          |Botania                       |botania                       |1.16.5-420.1        |CREATE_REG|Manifest: NOSIGNATURE
		curios-forge-1.16.5-4.0.5.3.jar                   |Curios API                    |curios                        |1.16.5-4.0.5.3      |CREATE_REG|Manifest: NOSIGNATURE
		Patchouli-1.16.4-53.2.jar                         |Patchouli                     |patchouli                     |1.16.4-53.2         |CREATE_REG|Manifest: NOSIGNATURE
		elevatorid-1.16.5-1.7.13.jar                      |Elevator Mod                  |elevatorid                    |1.16.5-1.7.13       |CREATE_REG|Manifest: NOSIGNATURE
		ftb-ultimine-forge-1605.3.1-build.45.jar          |FTB Ultimine                  |ftbultimine                   |1605.3.1-build.45   |CREATE_REG|Manifest: NOSIGNATURE
		tombstone-6.6.3-1.16.5.jar                        |Corail Tombstone              |tombstone                     |6.6.3               |CREATE_REG|Manifest: NOSIGNATURE
		logisticslasers-1.0.8.jar                         |Logistics Lasers              |logisticslasers               |1.0.8               |CREATE_REG|Manifest: NOSIGNATURE
		architectury-1.28.48.jar                          |Architectury                  |architectury                  |1.28.48             |CREATE_REG|Manifest: NOSIGNATURE
		cc-tweaked-1.16.5-1.100.1.jar                     |CC: Tweaked                   |computercraft                 |1.100.1             |CREATE_REG|Manifest: NOSIGNATURE
		observerlib-1.16.5-1.5.3.jar                      |ObserverLib                   |observerlib                   |1.16.5-1.5.3        |CREATE_REG|Manifest: NOSIGNATURE
		light-overlay-5.8.1.jar                           |Light Overlay                 |lightoverlay                  |5.8.1               |CREATE_REG|Manifest: NOSIGNATURE
		cloth-config-4.13.49-forge.jar                    |Cloth Config v4 API           |cloth-config                  |4.13.49             |CREATE_REG|Manifest: NOSIGNATURE
		bwncr-1.16.5-3.10.16.jar                          |Bad Wither No Cookie Reloaded |bwncr                         |1.16.5-3.10.16      |CREATE_REG|Manifest: NOSIGNATURE
		FastLeafDecay-v25.2.jar                           |FastLeafDecay                 |fastleafdecay                 |v25.2               |CREATE_REG|Manifest: NOSIGNATURE
		BetterAdvancements-1.16.5-0.1.1.115.jar           |Better Advancements           |betteradvancements            |0.1.1.115           |CREATE_REG|Manifest: NOSIGNATURE
		ClientTweaks_1.16.3-5.3.0.jar                     |Client Tweaks                 |clienttweaks                  |5.3.0               |CREATE_REG|Manifest: NOSIGNATURE
		Goo-1.16.5-0.8.3.8.jar                            |Goo                           |goo                           |0.8.3.8             |CREATE_REG|Manifest: NOSIGNATURE
		Cucumber-1.16.5-4.1.12.jar                        |Cucumber Library              |cucumber                      |4.1.12              |CREATE_REG|Manifest: NOSIGNATURE
		TrashSlot_1.16.3-12.2.1.jar                       |TrashSlot                     |trashslot                     |12.2.1              |CREATE_REG|Manifest: NOSIGNATURE
		InventorySpam-1.16.5-1.3.2.jar                    |Inventory Spam                |inventoryspam                 |1.3.2               |CREATE_REG|Manifest: NOSIGNATURE
		ftb-library-forge-1605.3.4-build.90.jar           |FTB Library                   |ftblibrary                    |1605.3.4-build.90   |CREATE_REG|Manifest: NOSIGNATURE
		snad-1.0.9-forge.jar                              |Snad                          |snad                          |1.0.9               |CREATE_REG|Manifest: NOSIGNATURE
		jei-1.16.5-7.7.1.139.jar                          |Just Enough Items             |jei                           |7.7.1.139           |CREATE_REG|Manifest: NOSIGNATURE
		shipping-pre-alpha-1.0.1.jar                      |Little Logistics              |shipping                      |pre-alpha-1.0.1     |ERROR     |Manifest: NOSIGNATURE
		WAWLA-1.16.5-8.0.1.jar                            |WAWLA                         |wawla                         |8.0.1               |CREATE_REG|Manifest: eb:c4:b1:67:8b:f9:0c:db:dc:4f:01:b1:8e:61:64:39:4c:10:85:0b:a6:c4:c7:48:f0:fa:95:f2:cb:08:3a:e5
		Clumps-6.0.0.27.jar                               |Clumps                        |clumps                        |6.0.0.27            |CREATE_REG|Manifest: NOSIGNATURE
		journeymap-1.16.5-5.7.3.jar                       |Journeymap                    |journeymap                    |5.7.3               |CREATE_REG|Manifest: NOSIGNATURE
		appliedenergistics2-8.4.6.jar                     |Applied Energistics 2         |appliedenergistics2           |8.4.6               |CREATE_REG|Manifest: 95:58:cc:83:9d:a8:fa:4f:e9:f3:54:90:66:61:c8:ae:9c:08:88:11:52:52:df:2d:28:5f:05:d8:28:57:0f:98
		NaturesCompass-1.16.5-1.9.1-forge.jar             |Nature's Compass              |naturescompass                |1.16.5-1.9.1-forge  |CREATE_REG|Manifest: NOSIGNATURE
		SimpleStorageNetwork-1.16.5-1.5.1.jar             |Simple Storage Network        |storagenetwork                |1.16.5-1.5.1        |CREATE_REG|Manifest: 1f:47:ac:b1:61:82:96:b8:47:19:16:d2:61:81:11:60:3a:06:4b:61:31:56:7d:44:31:1e:0c:6f:22:5b:4c:ed
		DungeonCrawl-1.16.5-2.3.5.jar                     |Dungeon Crawl                 |dungeoncrawl                  |2.3.5               |CREATE_REG|Manifest: NOSIGNATURE
		charginggadgets-1.3.0.jar                         |Charging Gadgets              |charginggadgets               |1.3.0               |CREATE_REG|Manifest: NOSIGNATURE
		pneumaticcraft-repressurized-1.16.5-2.14.7-283.jar|PneumaticCraft: Repressurized |pneumaticcraft                |1.16.5-2.14.7-283   |CREATE_REG|Manifest: NOSIGNATURE
		mcjtylib-1.16-5.1.3.jar                           |McJtyLib                      |mcjtylib                      |1.16-5.1.3          |CREATE_REG|Manifest: NOSIGNATURE
		rftoolsbase-1.16-2.1.2.jar                        |RFToolsBase                   |rftoolsbase                   |1.16-2.1.2          |CREATE_REG|Manifest: NOSIGNATURE
		xnet-1.16-3.0.16.jar                              |XNet                          |xnet                          |1.16-3.0.16         |CREATE_REG|Manifest: NOSIGNATURE
		rftoolsstorage-1.16-2.0.16.jar                    |RFToolsStorage                |rftoolsstorage                |1.16-2.0.16         |CREATE_REG|Manifest: NOSIGNATURE
		rftoolscontrol-1.16-4.0.14.jar                    |RFToolsControl                |rftoolscontrol                |1.16-4.0.14         |CREATE_REG|Manifest: NOSIGNATURE
		ToastControl-1.16.5-4.4.0.jar                     |Toast Control                 |toastcontrol                  |4.4.0               |CREATE_REG|Manifest: NOSIGNATURE
		rftoolspower-1.16-3.0.12.jar                      |RFToolsPower                  |rftoolspower                  |1.16-3.0.12         |CREATE_REG|Manifest: NOSIGNATURE
		ImmersivePetroleum-1.16.5-3.3.0-10.jar            |Immersive Petroleum           |immersivepetroleum            |3.3.0-10            |CREATE_REG|Manifest: NOSIGNATURE
		forge-1.16.5-36.2.22-universal.jar                |Forge                         |forge                         |36.2.22             |CREATE_REG|Manifest: 22:af:21:d8:19:82:7f:93:94:fe:2b:ac:b7:e4:41:57:68:39:87:b1:a7:5c:c6:44:f9:25:74:21:14:f5:0d:90
		BloodMagic-1.16.4-3.1.7-27.jar                    |Blood Magic                   |bloodmagic                    |1.16.4-3.1.7-27     |CREATE_REG|Manifest: NOSIGNATURE
		MysticalAgriculture-1.16.5-4.2.6.jar              |Mystical Agriculture          |mysticalagriculture           |4.2.6               |CREATE_REG|Manifest: NOSIGNATURE
		MysticalAgradditions-1.16.5-4.2.4.jar             |Mystical Agradditions         |mysticalagradditions          |4.2.4               |CREATE_REG|Manifest: NOSIGNATURE
		ironchest-1.16.5-11.2.21.jar                      |Iron Chests                   |ironchest                     |1.16.5-11.2.21      |CREATE_REG|Manifest: NOSIGNATURE
		CraftingTweaks_1.16.5-12.2.1.jar                  |Crafting Tweaks               |craftingtweaks                |12.2.1              |CREATE_REG|Manifest: NOSIGNATURE
		ZeroCore2-1.16.5-2.1.9.jar                        |Zero CORE 2                   |zerocore                      |1.16.5-2.1.9        |CREATE_REG|Manifest: NOSIGNATURE
		ExtremeReactors2-1.16.5-2.0.37.jar                |Extreme Reactors              |bigreactors                   |1.16.5-2.0.37       |CREATE_REG|Manifest: NOSIGNATURE
		forge-1.16.5-36.2.22-server.jar                   |Minecraft                     |minecraft                     |1.16.5              |CREATE_REG|Manifest: NOSIGNATURE
		cofh_core-1.16.5-1.4.0.6.jar                      |CoFH Core                     |cofh_core                     |1.4.0.6             |CREATE_REG|Manifest: 75:0b:cc:9b:64:2e:9b:c4:41:d1:95:00:71:ee:87:1a:b3:5e:4b:da:8e:e8:39:00:fd:5d:e5:9c:40:42:33:09
		thermal_foundation-1.16.5-1.4.1.7.jar             |Thermal Series                |thermal                       |1.4.1.7             |CREATE_REG|Manifest: 75:0b:cc:9b:64:2e:9b:c4:41:d1:95:00:71:ee:87:1a:b3:5e:4b:da:8e:e8:39:00:fd:5d:e5:9c:40:42:33:09
		thermal_expansion-1.16.5-1.4.1.3.jar              |Thermal Expansion             |thermal_expansion             |1.4.1.3             |CREATE_REG|Manifest: 75:0b:cc:9b:64:2e:9b:c4:41:d1:95:00:71:ee:87:1a:b3:5e:4b:da:8e:e8:39:00:fd:5d:e5:9c:40:42:33:09
		TConstruct-1.16.5-3.3.2.324.jar                   |Tinkers' Construct            |tconstruct                    |3.3.2.324           |CREATE_REG|Manifest: NOSIGNATURE
		WailaHarvestability-mc1.16.x-forge-1.1.15.jar     |Waila Harvestability          |wailaharvestability           |1.1.15              |CREATE_REG|Manifest: NOSIGNATURE
		rftoolsutility-1.16-3.1.8.jar                     |RFToolsUtility                |rftoolsutility                |1.16-3.1.8          |CREATE_REG|Manifest: NOSIGNATURE
		EnchantmentDescriptions-1.16.5-7.0.18.jar         |EnchantmentDescriptions       |enchdesc                      |7.0.18              |CREATE_REG|Manifest: eb:c4:b1:67:8b:f9:0c:db:dc:4f:01:b1:8e:61:64:39:4c:10:85:0b:a6:c4:c7:48:f0:fa:95:f2:cb:08:3a:e5
		swingthroughgrass-1.16.4-1.5.3.jar                |SwingThroughGrass             |swingthroughgrass             |1.16.4-1.5.3        |CREATE_REG|Manifest: NOSIGNATURE
		MouseTweaks-2.14-mc1.16.2.jar                     |Mouse Tweaks                  |mousetweaks                   |2.14                |CREATE_REG|Manifest: NOSIGNATURE
		titanium-1.16.5-3.2.8.4-10.jar                    |Titanium                      |titanium                      |3.2.8.4             |CREATE_REG|Manifest: NOSIGNATURE
		Psi 1.16-97.jar                                   |Psi                           |psi                           |1.16-96             |CREATE_REG|Manifest: NOSIGNATURE
		ImmersiveEngineering-1.16.5-5.0.7-143.jar         |Immersive Engineering         |immersiveengineering          |1.16.5-5.0.7-143    |CREATE_REG|Manifest: 44:39:94:cf:1d:8c:be:3c:7f:a9:ee:f4:1e:63:a5:ac:61:f9:c2:87:d5:5b:d9:d6:8c:b5:3e:96:5d:8e:3f:b7
		Ding-1.16.5-1.3.0.jar                             |Ding                          |ding                          |1.3.0               |CREATE_REG|Manifest: NOSIGNATURE
		rftoolsbuilder-1.16-3.1.5.jar                     |RFToolsBuilder                |rftoolsbuilder                |1.16-3.1.5          |CREATE_REG|Manifest: NOSIGNATURE
		jeiintegration_1.16.5-7.0.1.15.jar                |JEI Integration               |jeiintegration                |7.0.1.15            |CREATE_REG|Manifest: NOSIGNATURE
		flywheel-1.16-0.2.5.jar                           |Flywheel                      |flywheel                      |1.16-0.2.5          |CREATE_REG|Manifest: NOSIGNATURE
		create-mc1.16.5_v0.3.2g.jar                       |Create                        |create                        |v0.3.2g             |CREATE_REG|Manifest: NOSIGNATURE
		Mantle-1.16.5-1.6.152.jar                         |Mantle                        |mantle                        |1.6.152             |CREATE_REG|Manifest: NOSIGNATURE
		rats-7.2.0-1.16.5.jar                             |Rats                          |rats                          |7.2.0               |CREATE_REG|Manifest: NOSIGNATURE
		ftb-backups-2.1.2.2.jar                           |FTB Backups                   |ftbbackups                    |2.1.2.2             |CREATE_REG|Manifest: NOSIGNATURE
		recycler-1605.1.0-build.11.jar                    |Item Recycler                 |recycler                      |1605.1.0-build.11   |CREATE_REG|Manifest: NOSIGNATURE
		AutoRegLib-1.6-49.jar                             |AutoRegLib                    |autoreglib                    |1.6-49              |CREATE_REG|Manifest: NOSIGNATURE
		Quark-r2.4-321.jar                                |Quark                         |quark                         |r2.4-321            |CREATE_REG|Manifest: NOSIGNATURE
		StorageDrawers-1.16.3-8.5.0.jar                   |Storage Drawers               |storagedrawers                |8.5.0               |CREATE_REG|Manifest: NOSIGNATURE
		performant-1.16.2-5-3.82m.jar                     |Performant                    |performant                    |3.73m               |CREATE_REG|Manifest: NOSIGNATURE
		appleskin-forge-mc1.16.x-2.2.0.jar                |AppleSkin                     |appleskin                     |mc1.16.4-2.2.0      |CREATE_REG|Manifest: NOSIGNATURE
	Crash Report UUID: 512ddbf7-4963-499d-a04f-8bd1cb31728c
```

**Comment by MuradAkh** [comment]
Thanks for reporting, looks like I messed up client/server separation somewhere in initialisation, should be an easy fix.

**Comment by ZL1LAC** [comment]
No problem let me know how else I can help.

**Comment by MuradAkh** [comment]
fixed in #d91c1ecba3403ba227a29462fff5765a03e1ff16
